keps/0009-operator-toolkit.md

```diff
@@ -119,46 +119,79 @@ maintainers:
   - Alice <alice@example.com>
 url: https://github.com/myoperator/myoperator
 tasks:
-  deploy:
-    resources:
-      - pvc.yaml
-      - deployment.yaml
-      - service.yaml
-      - deployment2.yaml
-      - job.yaml
-    patches:
```

Comment:
I'm not sure what this `patches` and `patchesStrategicMerge` semantic is. We never supported it, but might support it in the future?

Reply:
Yeah, these are Kustomize constructs.